less/variables.less

```diff
@@ -11,4 +11,29 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+// MY CODE STARTS HERE
+
+@mobile: ~'(max-width: 500px)';
+
+ @media @mobile {
```

Comment:
We talked about this in standup, but don't forget- we actually use these media queries inside the other files. This media query would exist inside `navigation.less` '' '